types/src/account_config/resources/account.rs

```diff
@@ -24,6 +24,7 @@ pub struct AccountResource {
     sent_events: EventHandle,
     sequence_number: u64,
     is_frozen: bool,
+    role_id: u64,
```

Comment:
curious why we don't just use an enum and the magic of lcs or serde?

Reply:
I think that works  if `role_id` is a u8, but will it work for u64 too?